app/controllers/api/github_issues_controller.rb

```diff
@@ -3,11 +3,13 @@ class Api::GithubIssuesController < ApplicationController
   before_action :create_github_client, only: [:index, :create]
 
   def index
-    @issues = @client.issues.list
+    @issues = @client.issues.list user: 'TeamMartinez', repo: 'YOLO-api', state: 'open'
     render json: @issues
   end
 
   def create
+    # TODO: make user: and repo: params default somewhere and use params
+    @client.issues.create user: 'TeamMartinez', repo: 'YOLO-api', title: params[:title]
     render json: { errors: "not implemented" }
```

Comment:
The only thing is that `:require` seems to behave differently than `:permit` so we will need to investigate how to require what we want. I also dont know if `:permit` and `:require` are rails specific things for `params` or if there is general documentation on filtering arrays